code/style.css

```diff
@@ -1,7 +1,7 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: grey;
+  --secondary: black;
 }
```

Comment:
Love your project! really great, do you mind I share your netlify link with my niece and nephew? think they'll love it